### PR TITLE
fix the subresource operation name when overriding path

### PR DESCRIPTION
**Pull request by bendavies**
This operation name was wrong.
The prefix must be removed when overriding a path

### CODE REVIEW

File: core/subresources.md [hunk 1]
```diff
@@ -207,7 +207,7 @@ You can control the path of subresources with the `path` option of the `subresou
  * ...
  * @ApiResource(
  *      subresourceOperations={
- *          "api_questions_answer_get_subresource"={
+ *          "answer_get_subresource"={
```
Comment (soyuka) [inline core/subresources.md]: this looks buggy, it should be the route name you get under `bin/console debug:router`, we definitely need to improve the subresources DX.

Comment (AdamKasp) [reply core/subresources.md:210]: information from lines https://github.com/api-platform/docs/pull/1071/files#diff-3a44ca00264369c7b87966b5596aeb33R194-R196
should also be updated, for now, it is wrong information.

**Comment by dunglas** [comment]
@soyuka what about this PR? Should we close it?

**Comment by soyuka** [comment]
It's not resolved I guess we can close it and related on https://github.com/api-platform/core/issues/3458 instead while this isn't fixed...

**Comment by bendavies** [comment]
as far as i can remember the docs are incorrect so this was good to merge?

**Comment by soyuka** [comment]
I do have the `api_questions_answer_get_subresource` name on my `bin/console` debug:router name

**Pull request closed without merging**